libethcore/BlockInfo.cpp

```diff
@@ -212,9 +212,10 @@ void BlockInfo::verifyInternals(bytesConstRef _block) const
 	if (transactionsRoot != t.root())*/
 	auto txList = root[1];
 	auto expectedRoot = trieRootOver(txList.itemCount(), [&](unsigned i){ return rlp(i); }, [&](unsigned i){ return txList[i].data(); });
+	cnote << "expect trroot " << toString(expectedRoot);
```

Comment:
either remove additional messages or place them at a higher level than cnote and reformat them (e.g. `"Expected root:"`).